.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "livePreview.defaultPreviewPath": "/notes.html"
```

Comment:
remove this change it's not related to the project

Reply:
Removed